_pages/Dashboard/index.js

```diff
@@ -1,5 +1,33 @@
+import { useState, useEffect } from 'react';
+import axios from 'axios';
+
 const Dashboard = () => {
-  return <h1>Dashboard</h1>;
+  const [player, setPlayer] = useState([]);
+
+  const getPlayerInfo = async () => {
+    const response = await axios.get('/api/identity');
+    setPlayer(response.data);
+  };
+
+  useEffect(() => {
+    getPlayerInfo();
+  }, []);
+
+  console.log(player);
+
+  return (
+    <div className="player-container" style={{ color: 'red' }}>
+      <h1>Dashboard</h1>
+      {player.map((player) => (
```

Comment:
Another good naming rule of thumb: when using `.map()`, the left-hand-side collection should usually have a pluralized name (e.g., `players`). That way you can use the singular (e.g., `player`) as your callback argument name, which produces really clean, clear and easy to understand code (each element in `players` is a `player`) 👌🏻

Reply:
This makes sense :) thanks